samples/stock/indicators/disparity-index/demo.js

```diff
@@ -0,0 +1,44 @@
+Highcharts.getJSON('https://demo-live-data.highcharts.com/aapl-ohlc.json', function (data) {
+    Highcharts.stockChart('container', {
+
+        legend: {
+            enabled: true
+        },
+
+        plotOptions: {
+            series: {
+                showInLegend: true
+            }
+        },
+
+        rangeSelector: {
+            selected: 1
+        },
+
+        title: {
+            text: 'AAPL Stock Price'
+        },
+
+        yAxis: [{
+            height: '60%'
+        }, {
+            height: '40%',
+            top: '60%'
+        }],
+
+        series: [{
+            type: 'candlestick',
+            id: 'aapl',
+            name: 'AAPL Stock Price',
+            data: data
+        }, {
+            type: 'disparityindex',
+            linkedTo: 'aapl',
+            yAxis: 1,
+            params: {
+                period: 9
+            }
+        }]
+
```

Comment:
Unnecessary space

Reply:
Do you mean "enter"? Line break? Is there a 2-lines break or a 1-line break? If 1-line, then it is intended.

Reply:
yep- 1-line break- I accept that it might be intentional ;)